crates/hyperqueue/src/client/commands/autoalloc.rs

```diff
@@ -306,7 +309,12 @@ wasted allocation duration."
             "--cpus".to_string(),
             format!("\"{}\"", cpus.into_original_input()),
         ]);
+    } else {
+        log::warn!(
+            "Creating an autoalloc queue without specifying worker core count. Consider specifying the core count of workers in this queue to improve the efficiency of automatic allocation."
```

Comment:
I would formulate it rather something like:
"No resources are specified for the queue. Autoallocator is running in a probing mode until a first worker
occur and we get the information about worker resources"